src/Security/Authentication/JwtBearer/src/JwtBearerEvents.cs

```diff
@@ -32,6 +36,7 @@ public class JwtBearerEvents
         public Func<JwtBearerChallengeContext, Task> OnChallenge { get; set; } = context => Task.CompletedTask;
 
         public virtual Task AuthenticationFailed(AuthenticationFailedContext context) => OnAuthenticationFailed(context);
+        public virtual Task AuthorizationFailed(AuthenticationFailedContext context) => OnAuthorizationFailed(context);
```

Comment:
Empty line between methods.
```suggestion

        public virtual Task AuthorizationFailed(AuthenticationFailedContext context) => OnAuthorizationFailed(context);
```

Reply:
Corrected